Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -186,6 +187,33 @@ void ProjectData::buildProcesses()
                     *_mesh_vec[0], *nl_slv, std::move(time_disc),
                     std::move(conv_crit), _process_variables, _parameters, pc));
         }
+        else if (type == "SMALL_DEFORMATION")
+        {
+            switch (pc.getConfigParameter<int>("dimension"))
```

Comment:
In general all possibilities can occur, like 2D displacement in 3D, or 2D movement of line elements, but the non-matching dimensions needs special models.
For now, indeed the displacement and the mesh dimensions are equal.
@nagelt Can you comment on whether we need that flexibility?